Fix build error with rustc 1.95 in f45

error[E0425]: cannot find type `LaneCount` in module `core::simd`
   --> ../../third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs:234:15
    |
234 |   core::simd::LaneCount<N>: core::simd::SupportedLaneCount,
    |               ^^^^^^^^^ not found in `core::simd`
error[E0405]: cannot find trait `SupportedLaneCount` in module `core::simd`
   --> ../../third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs:234:41
    |
234 |   core::simd::LaneCount<N>: core::simd::SupportedLaneCount,
    |                                         ^^^^^^^^^^^^^^^^^^ not found in `core::simd`
error[E0425]: cannot find type `LaneCount` in module `core::simd`
   --> ../../third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs:155:15
    |
155 |   core::simd::LaneCount<N>: core::simd::SupportedLaneCount,
    |               ^^^^^^^^^ not found in `core::simd`
error[E0405]: cannot find trait `SupportedLaneCount` in module `core::simd`
   --> ../../third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs:155:41
    |
155 |   core::simd::LaneCount<N>: core::simd::SupportedLaneCount,


diff -up chromium-147.0.7727.116/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs.me chromium-147.0.7727.116/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs
--- chromium-147.0.7727.116/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs.me	2026-04-26 13:19:28.648591553 +0200
+++ chromium-147.0.7727.116/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs	2026-04-26 13:23:22.967926065 +0200
@@ -143,25 +143,7 @@ impl_unsafe_marker_for_simd!(
   }
 );
 
-#[rustversion::before(2026-01-27)] // See https://github.com/Lokathor/bytemuck/issues/343
 #[cfg(feature = "nightly_portable_simd")]
-#[cfg_attr(
-  feature = "nightly_docs",
-  doc(cfg(feature = "nightly_portable_simd"))
-)]
-unsafe impl<T, const N: usize> Pod for core::simd::Simd<T, N>
-where
-  T: core::simd::SimdElement + Pod,
-  core::simd::LaneCount<N>: core::simd::SupportedLaneCount,
-{
-}
-
-#[rustversion::since(2026-01-27)] // See https://github.com/Lokathor/bytemuck/issues/343
-#[cfg(feature = "nightly_portable_simd")]
-#[cfg_attr(
-  feature = "nightly_docs",
-  doc(cfg(feature = "nightly_portable_simd"))
-)]
 unsafe impl<T, const N: usize> Pod for core::simd::Simd<T, N>
 where
   T: core::simd::SimdElement + Pod,
diff -up chromium-147.0.7727.116/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs.me chromium-147.0.7727.116/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs
--- chromium-147.0.7727.116/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs.me	2026-04-26 12:54:16.801715156 +0200
+++ chromium-147.0.7727.116/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs	2026-04-26 13:23:50.948183197 +0200
@@ -222,25 +222,7 @@ impl_unsafe_marker_for_simd!(
     }
 );
 
-#[rustversion::before(2026-01-27)] // See https://github.com/Lokathor/bytemuck/issues/343
 #[cfg(feature = "nightly_portable_simd")]
-#[cfg_attr(
-  feature = "nightly_docs",
-  doc(cfg(feature = "nightly_portable_simd"))
-)]
-unsafe impl<T, const N: usize> Zeroable for core::simd::Simd<T, N>
-where
-  T: core::simd::SimdElement + Zeroable,
-  core::simd::LaneCount<N>: core::simd::SupportedLaneCount,
-{
-}
-
-#[rustversion::since(2026-01-27)] // See https://github.com/Lokathor/bytemuck/issues/343
-#[cfg(feature = "nightly_portable_simd")]
-#[cfg_attr(
-  feature = "nightly_docs",
-  doc(cfg(feature = "nightly_portable_simd"))
-)]
 unsafe impl<T, const N: usize> Zeroable for core::simd::Simd<T, N>
 where
   T: core::simd::SimdElement + Zeroable,
